contrib/inventory/apache-libcloud.py

```diff
@@ -262,7 +262,7 @@ def get_host_info(self):
             # Handle complex types
             if type(value) in [int, bool]:
                 instance_vars[key] = value
-            elif type(value) in [str, unicode]:
+            elif type(value) in string_types:
```

Comment:
Need to change this to an isinstance check.  Otherwise on python2 the conditional will be equivalent to:
```
elif str in (basetring,)
```

which will be False.